packages/components/src/__future__/ExampleComponent/docs/ExampleComponent.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react"
+import { ComponentMeta, ComponentStory } from "@storybook/react"
+import { ExampleComponent } from "../ExampleComponent"
+
+export default {
+  title: "AIO/future/ExampleComponent (Do Not Use)",
+  component: ExampleComponent,
+  parameters: {},
+} as ComponentMeta<typeof ExampleComponent>
+
+export const DefaultStory: ComponentStory<typeof ExampleComponent> = args => (
+  <ExampleComponent />
+)
```

Comment:
Probably for the other one too. Although this is nitpicking I guess.

```suggestion
export const DefaultStory: ComponentStory<typeof ExampleComponent> = () => (
  <ExampleComponent />
)
```
or
```suggestion
export const DefaultStory: ComponentStory<typeof ExampleComponent> = args => (
  <ExampleComponent {...args} />
)
```